### PR TITLE
[109] The Software Developer Conditionals - Frontend 

### PR DESCRIPTION
**Pull request by Onyinye101**
## Task: `[109] The Software Developer Conditionals`
- `Created softwaredeveloper.js`

### CODE REVIEW

**Review by BenjaminKamena — CHANGES_REQUESTED** [review]
Convert positiveDifference to an absolute number, use Math.ab to do that

**Comment by Onyinye101** [comment]
Why?

**Comment by BenjaminKamena** [comment]
The criteria is saying absolute number right?

**Comment by BenjaminKamena** [comment]
Ohh sorry its saying positive number, so need to change it to positive number by using Math.ab

File: submissions/onyinyeChukwuka/javascript/softwaredeveloper.js [hunk 1]
```diff
@@ -0,0 +1,12 @@
+const positiveDifference = 123 - 7;
```
Comment (TundeOladejo) [inline submissions/onyinyeChukwuka/javascript/softwaredeveloper.js:1]: Use `Math.abs` to return an absolute value

**Review by TundeOladejo — CHANGES_REQUESTED** [review]

**Comment by TundeOladejo** [comment]
> Ohh sorry its saying positive number, so need to change it to positive number by using Math.ab

Nice one @BenjaminKamena. It's `Math.abs` not `Math.ab`

**Review by adaugochi — CHANGES_REQUESTED** [review]
@Onyinye101 

You might be wondering why you are asked to use `Math.abs`. What that `Math` function or method does is to return the absolute value of a number. E.g if you do `2 - 5` and you make use of the `Math.abs` method you will get `3` instead of `-3`. 

So your task says `positive difference between 7 and 123`. Meaning when you so `7 - 123` even when you know it will give you a negative value, you still want to get a positive value. I don't know if you understand? 

Instead of you doing this `const positiveDifference = 123 - 7;` like what most of you did, the best would have been this `const positiveDifference = Math.abs(7 - 123)`;

**Comment by Onyinye101** [comment]
Thank you so much for this. It’s very insightful. Thank you again.

**Review by adaugochi — APPROVED** [review]

**Pull request merged**